examples/docker-build/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:latest
+
+# Environment variables
+ENV PIXI_VERSION=latest
+ENV INSTALL_DIR=/usr/local/bin
+ENV REPO=prefix-dev/pixi
+ENV PLATFORM=unknown-linux-musl
+ENV ARCH=x86_64
+ENV PROJECT_NAME=pixi-in-docker
+
+# Install curl and tar
+RUN apt-get update && \
+    apt-get install -y curl tar
+
+# Download and install pixi
+RUN if [ "$PIXI_VERSION" = "latest" ]; then \
+      DOWNLOAD_URL="https://github.com/$REPO/releases/latest/download/pixi-$ARCH-$PLATFORM.tar.gz"; \
+    else \
+      DOWNLOAD_URL="https://github.com/$REPO/releases/download/$PIXI_VERSION/pixi-$ARCH-$PLATFORM.tar.gz"; \
```

Comment:
```suggestion
      DOWNLOAD_URL="https://github.com/$REPO/releases/latest/download/pixi-$(uname -m)-$PLATFORM.tar.gz"; \
    else \
      DOWNLOAD_URL="https://github.com/$REPO/releases/download/$PIXI_VERSION/pixi-$(uname -m)-$PLATFORM.tar.gz"; \
```

Reply:
Good tip! Changed it manually in https://github.com/prefix-dev/pixi/pull/353/commits/56a4e25e2e039944acb01164a0514481ffdbc4bd